internal/log/field/fields.go

```diff
@@ -47,6 +47,12 @@ func Coordinate(coordinate coordinate.Coordinate) Field {
 		}}
 }
 
+// Type builds a Field containing information about a config type. This is used in cases where no full coordinate exists,
+// but only a config type is known - for example in download or deletion
+func Type(t string) Field {
```

Comment:
As they all add one `field` I find `field.Type(...)` or `field.Environment` reads pretty nice and clearly.
If they added several `fields` based on some input I'd agree that `fields.From...` seems better, but no current method adds more than one `field`.

(Discussing based on how it reads because nitpick)